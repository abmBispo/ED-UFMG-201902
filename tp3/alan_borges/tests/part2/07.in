16
e por que e tambem por que e que seria preso e quem e que me
c e
c que
c por
c tambem
c me
c preso
c quem
c seria
