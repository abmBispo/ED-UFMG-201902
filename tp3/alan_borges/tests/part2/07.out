11
01
001
000
1000
1001
1010
1011
